CONTRIBUTING.md

```diff
@@ -33,12 +33,13 @@ investigated. Bug reports should have:
 Pull requests are the best way to propose code changes:
 
 1. Fork the repo and create your branch from `master`.
-2. Add tests, if appropriate.
-3. If you add or change code generator command line options, update the
+2. Run `./examples/run_djinni.sh` and check in any changes to generated code.
```

Comment:
If you modified the code generator, run `run_djinni.sh` in `./examples`, `./test-suite`, `./perftest` and check in any changes to generated code.

Reply:
Done